contribute/index.markdown

```diff
@@ -0,0 +1,11 @@
+---
+layout: std
+title: Enter the void - Contribute
+---
+
+<div>
+<h2>Contribute</h2>
```

Comment:
While contribute is a decent title, I suggest that this heading be a call to action.  Something like "Join Us" or "Be a Part of Void" or something like that.

Reply:
Sure, i'll change it to "Join Us", which is nice and short.

Reply:
Also, should the path be changed from `/contribute/` to `/joinus/`?

Reply:
the path can stay as is, many web browsers don't even show full paths anymore.